ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/nodes/imm_ukf_pda/imm_ukf_pda.cpp

```diff
@@ -665,131 +546,278 @@ void ImmUkfPda::staticClassification()
 {
   for (size_t i = 0; i < targets_.size(); i++)
   {
-    if (!targets_[i].is_static_ && targets_[i].tracking_num_ == TrackingState::Stable &&
-        targets_[i].lifetime_ > life_time_thres_)
+    targets_[i].vel_history_.push_back(targets_[i].x_merge_(2));
+    if (targets_[i].tracking_num_ == TrackingState::Stable && targets_[i].lifetime_ > life_time_thres_)
     {
-      if ((targets_[i].dist_from_init_ < static_distance_thres_) &&
-          (targets_[i].mode_prob_rm_ > targets_[i].mode_prob_cv_ ||
-           targets_[i].mode_prob_rm_ > targets_[i].mode_prob_ctrv_))
+      double sum_vel = 0;
+      double avg_vel = 0;
+      for (int ind = 1; ind < life_time_thres_; ind++)
+      {
+        sum_vel += targets_[i].vel_history_.end()[-ind];
```

Comment:
end()[-ind] is not common style. It's better to use rbegin() and rend()

Reply:
This pr aims to change topic type.
Definitely modifying in next pr.